web/ui/react-app/package.json

```diff
@@ -64,9 +64,12 @@
     "@types/flot": "0.0.31",
     "@types/moment-timezone": "^0.5.10",
     "@types/reactstrap": "^8.0.5",
+    "@types/sinon": "^7.5.0",
```

Comment:
Nothing in this file changed except for alphabetic sorting of packages. Does anyone know if this depends on the yarn versions or something? Would be good to get this consistent for everyone so we don't get extra diff noise all the time.

Reply:
Ah sorry, `jest-fetch-mock` is new after all, but the rest just got reordered. My comment about that still stands.